rules/app.bzl

```diff
@@ -59,9 +60,9 @@ def ios_application(name, apple_library = apple_library, infoplists_by_build_set
     application_kwargs["families"] = application_kwargs.pop("families", ["iphone", "ipad"])
 
     force_load_name = name + ".force_load_direct_deps"
-    force_load_direct_deps(name = force_load_name, deps = library.lib_names, tags = ["manual"])
-    default_deps = [force_load_name] + library.lib_names
+    force_load_direct_deps(name = force_load_name, deps = kwargs.get("deps"), tags = ["manual"])
```

Comment:
Do we still need to add library.lib_names in the deps?

Reply:
No it needs the actual deps vs the `lib_names` in order to compute force loading given dynamic frameworks